docs/source/optimisation.rst

```diff
@@ -92,20 +92,28 @@ SIRT
    :members: update, update_objective
    :inherited-members: run, update_objective_interval, max_iteration
 
-ISTA
+ISTA/PGD
```

Comment:
If we add the alias instead of a new class it may be simpler to just add a simple line explaining that PGD is an alias of ISTA.

```suggestion
ISTA/PGD

ISTA is also known as PGD... etc
```